Sources/EpoxyCore/SwiftUI/LayoutUtilities/SwiftUIMeasurementContainer.swift

```diff
@@ -413,16 +455,16 @@ extension ViewType {
     #endif
   }
 
-  /// Whether this view or any of its subviews has a subview that has a double layout pass `UILabel`
-  /// as determined by a non-zero `preferredMaxLayoutWidth`, which implies that it should get a
-  /// `intrinsicHeightProposedWidth` sizing strategy to allow the label to wrap and grow.
+  /// Whether this view or any of its subviews has a subview that has a double layout pass `UILabel` as determined by being
+  /// configured to show multiple lines of text. This view should get a `intrinsicHeightProposedOrIntrinsicWidth` sizing
+  /// strategy so that it wraps correctly.
   @nonobjc
   fileprivate func containsDoubleLayoutPassSubviews() -> Bool {
     #if os(macOS)
     return false
     #else
     var contains = false
-    if let label = self as? UILabel, label.preferredMaxLayoutWidth > 0 {
+    if let label = self as? UILabel, label.numberOfLines != 1 {
```

Comment:
We can simply check if the label is configured to show multiple lines. No need depending on an extra layout pass to determine if the label can wrap.